CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v0.3.0
+
+## Changes
+- [[#40]](https://github.com/rust-vmm/seccompiler/pull/40): Update Rust
+  to Edition 2021
+
+## Fixes
+
+- [[#31]](https://github.com/rust-vmm/seccompiler/issues/31): Implement
+  From<BackendError> for Error
+- [[#40]](https://github.com/rust-vmm/seccompiler/pull/40): Fix clippy
+  complaints about missing Eq when PartialEq is implemented
```

Comment:
nit: enclose `From<BackendError> for Error`, `Eq` and `PartialEq` in code backtics so they show up as code